spec/files/pef-compat-elf.rb

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env ruby
```

Comment:
This file is complained by rubocop :p

Reply:
awww, that's a new ruby syntax i didn't test that :sweat: